### PR TITLE
Use RFC3339 timestamps for improved human readability 

### PR DESCRIPTION
**Pull request by mjlshen**
[OSD-15112](https://issues.redhat.com//browse/OSD-15112)

* Defaulting timestamps to RFC3339 instead of epoch time and removing misleading stack traces that can show up to match https://github.com/openshift/aws-vpce-operator/blob/main/main.go#LL78-L83C3
* go.mod wasn't formatted for whatever reason
* Ran `make ensure-fips` which might have gotten missed in a previous `make boilerplate-update`

The logs now look like this:
```json
{"level":"error","ts":"2023-02-06T21:44:55-05:00","logger":"setup","msg":"unable to get WatchNamespace,the manager will watch and manage resources in all namespaces","error":"WATCH_NAMESPACE must be set"}
{"level":"info","ts":"2023-02-06T21:44:55-05:00","logger":"leader","msg":"Trying to become the leader."}
{"level":"error","ts":"2023-02-06T21:44:55-05:00","logger":"setup","msg":"failed to setup leader lock","error":"namespace not found for current environment"}
```

### CODE REVIEW

**Review by geowa4 — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -87,7 +88,10 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	opts := zap.Options{
-		Development: true,
+		Development: false,
```
Comment (iamkirkbater) [inline main.go:91]: Is this change on purpose?  Going from false -> true?

Comment (mjlshen) [reply main.go:91]: It was on purpose since this would let the operator use production logging defaults. It could still be overridden when developing by using `--zap-devel=true`

https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/building-operators/golang/references/logging.md#default-zap-logger

**Review by boranx — COMMENTED** [review]
/lgtm

I'll look at this when it lands on the staging

**Comment by boranx** [comment]
testing results:

```shell
oc logs cloud-ingress-operator-8fd65b48-vhxvc -n openshift-cloud-ingress-operator | tail -n 3
{"level":"info","ts":"2023-02-08T13:50:02Z","logger":"controller_apischeme","msg":"Reconciling APIScheme","Request.Namespace":"openshift-cloud-ingress-operator","Request.Name":"rh-api"}
{"level":"info","ts":"2023-02-08T13:51:02Z","logger":"controller_apischeme","msg":"Reconciling APIScheme","Request.Namespace":"openshift-cloud-ingress-operator","Request.Name":"rh-api"}
{"level":"info","ts":"2023-02-08T13:52:03Z","logger":"controller_apischeme","msg":"Reconciling APIScheme","Request.Namespace":"openshift-cloud-ingress-operator","Request.Name":"rh-api"}
```
